util/file_splitter.go

```diff
@@ -1,15 +1,19 @@
 package util
 
 import (
+	"fmt"
 	"io"
 	"math"
 	"os"
 	"path/filepath"
 	"strconv"
 
+	"github.com/cloudfoundry-incubator/multiapps-cli-plugin/configuration"
 	"github.com/pborman/uuid"
 )
 
+const DefaultFileChunkCount = 50
```

Comment:
Rename to `MaxFileChunkCount`, since it is not configurable and we can send less, but we cannot send more.